Riot/Modules/Room/VoiceMessages/VoiceMessageAudioPlayer.swift

```diff
@@ -121,8 +121,9 @@ class VoiceMessageAudioPlayer: NSObject {
         audioPlayer?.seek(to: .zero)
     }
     
-    func seekToTime(_ time: TimeInterval) {
-        audioPlayer?.seek(to: CMTime(seconds: time, preferredTimescale: 60000))
+    func seekToTime(_ time: TimeInterval, completionHandler:@escaping (Bool) -> Void = { _ in }) {
+        isStopped = false
```

Comment:
It doesn't feel like it should be here. I should be able to scrub while the player is stopped without changing its playing state.

Reply:
If it's not set it breaks this code -> https://github.com/vector-im/element-ios/pull/4878/files#diff-5aece56f4eb2d66c6dac2cf14ca34006bdf850be104fa47ba5cacc7fbcf249f1R432

My thinking was by seeking to a specified point you are no longer `stopped` but are more `paused` at that point (if it is not currently playing). And if you press play it will play from that point and not the start.